frb_codegen/src/commands.rs

```diff
@@ -235,15 +298,14 @@ pub fn format_rust(path: &str) -> Result {
     Ok(())
 }
 
-pub fn format_dart(path: &str, line_length: i32) -> Result {
-    debug!(
-        "execute format_dart path={} line_length={}",
-        path, line_length
-    );
-    let res = call_shell(&format!(
-        "dart format {} --line-length {}",
-        path, line_length
-    ));
+pub fn format_dart(path: &[PathBuf], line_length: u32) -> Result {
+    debug!("execute format_dart line_length={}", line_length);
+    let mut args = args!("format", "--line-length", line_length.to_string());
+    args.extend(path.iter().cloned());
```

Comment:
nit: Would be great if it could be immutable

Hoping to exist some python-like syntax:

`vec!['format', '--line-length', line_length, *path]`